R/axis.R

```diff
@@ -153,7 +153,7 @@ dyAxis <- function(dygraph,
   attrs$axes[[axis$name]] <- axis$options  
   
   # return modified dygraph
-  dygraph$x$attrs <- attrs
+  dygraph$x$attrs <- mergeLists(dygraph$x$attrs, attrs)
```

Comment:
I'm now thinking that I might be wrong about needing to call mergeLists as this line of code starts things off by making a copy of `dygraph$x$attrs`: https://github.com/pshevtsov/rstudio-dygraphs/blob/feature/dygraphs-1.1/R/axis.R#L109

What happens if you remove the call to mergeLists and try to use the drawXAxis parameter in a dyOptions **followed by** calling dyAxis on the x axis? If it works as expected then we don't need the mergeLists.